spectrochempy/core/readers/read_omnic.py

```diff
@@ -702,35 +691,31 @@ def _read_spg(*args, **kwargs):
         offset=firstx[0],
         increment=spacing,
         size=int(nx[0]),
-        title=xtitles[0],
+        long_name=xtitles[0],
         units=xunits[0],
     )
 
     _y = Coord(
-        timestamps,
-        title="acquisition timestamp (GMT)",
-        units="s",
-        labels=(acquisitiondates, spectitles),
+        np.array(acquisitiondates),
+        long_name="acquisition date (GMT)",
```

Comment:
"acquisition date (GMT)" -> "acquisition date (UTC)"

Reply:
Yes I hesitated (because in fact GMT and UTC are not (quite) the same thing). But it is logical. Anyway it never displays this string because I modify it later, I have to find where.